addons/website_portal/tests/__init__.py

```diff
@@ -1 +1,5 @@
-import test_load_process
\ No newline at end of file
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import test_load_process
```

Comment:
My eyes are bleeding, we should not have two different kind of imports when having two lines of import.